frontend-react/README.md

```diff
@@ -43,11 +43,31 @@ for any error status codes.
 ```bash
 yarn # Will install all dependencies in package.json
 
-yarn start # Runs the React app
-yarn test # Runs the React app test suite
-yarn build # Builds the React app
+yarn start:localdev # Runs the React app
```

Comment:
line 26 under Serving will need this change too. i updated it manually when I was debugging with Seth earlier though so a rebase should catch this